test/GenServer.erl

```diff
@@ -0,0 +1,23 @@
+-module(test_genServer@foreign).
+
+-export([ startGprocFFI/0
+        , sleep/1
+        ]).
+
+%%% ----------------------------------------------------------------------------
+%%% FFI API
+%%% ----------------------------------------------------------------------------
+startGprocFFI() ->
+  fun() ->
+      %% TODO - not this, anything but this
+      code:add_patha("/Users/adrianroe/dev/rtsv2/_build/default/lib/gproc/ebin/"),
+      application:ensure_started(gproc),
+      unit
+  end.
```

Comment:
You can presumably now use the actual rebar3 eunit test runner with before/after actions to get this sort of thing working properly